artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/wireformat/SessionCreateConsumerMessage.java

```diff
@@ -25,20 +26,24 @@
 
    private SimpleString filterString;
 
+   private int priority;
```

Comment:
I will change it as `byte` if we are not planning to support priorities > 127 ie `Byte::MAX_VALUE`
For small messages it *could* save some space (TBH to be verified with JOL tool)

Reply:
We should support int, as AMQP uses can use -2^31 to 2^31-1. 

This changes nothing on the Message size, or any space concerns, this is Only on consumer creation.

Reply:
marking resolved.